frontendUiImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/GitMacheteErrorHandler.java

```diff
@@ -25,9 +41,73 @@ public boolean submit(
       @Nullable String additionalInfo,
       Component parentComponent,
       Consumer<? super SubmittedReportInfo> consumer) {
-    GuiUtils.invokeLaterIfNeeded(() -> BrowserUtil.browse(
-        "https://github.com/VirtusLab/git-machete-intellij-plugin/issues/new?assignees=&labels=bug&template=bug_report.md"),
-        ModalityState.NON_MODAL);
+    String reportBody = getReportBody(events, additionalInfo);
+
+    try {
+      val uri = constructNewGitHubIssueUri(reportBody);
+
+      GuiUtils.invokeLaterIfNeeded(() -> BrowserUtil.browse(uri), ModalityState.NON_MODAL);
+    } catch (URISyntaxException e) {
+      LOG.error("Cannot construct URI to open new bug issue! Erroneous report body: ${reportBody}", e);
+    }
     return true;
   }
+
+  private URI constructNewGitHubIssueUri(@Nullable String reportBody) throws URISyntaxException {
+    val uriBuilder = new URIBuilder("https://github.com/VirtusLab/git-machete-intellij-plugin/issues/new");
+    uriBuilder.addParameter("labels", "bug");
+    if (reportBody != null) {
+      uriBuilder.addParameter("body", reportBody);
+    } else {
+      uriBuilder.addParameter("template", "bug_report.md");
+    }
+    return uriBuilder.build();
+  }
+
+  private @Nullable String getReportBody(
+      IdeaLoggingEvent[] events,
+      @Nullable String additionalInfo) {
+    String reportBody = null;
+    try {
+      reportBody = getBugTemplate();
+      for (java.util.Map.Entry<String, String> entry : getTemplateVariables(events, additionalInfo).entrySet()) {
+        reportBody = reportBody.replace("%${entry.getKey()}%", entry.getValue());
+      }
+    } catch (IOException | AssertionError e) {
+      LOG.error("Error while creating bug report template", e);
+    }
+    return reportBody;
+  }
+
+  private String getBugTemplate() throws IOException, AssertionError {
+    val classLoader = GitMacheteErrorHandler.class.getClassLoader();
+    assert classLoader != null;
+    val is = classLoader.getResourceAsStream("bug_report.md");
+    assert is != null;
+    return IOUtils.toString(is, StandardCharsets.UTF_8);
+  }
+
+  private java.util.Map<String, String> getTemplateVariables(
+      IdeaLoggingEvent[] events,
+      @Nullable String additionalInfo) {
+    val sep = System.lineSeparator();
+    val templateVariables = new java.util.HashMap<String, String>();
+    // Ide version, ie. Intellij Community 2021.3.1
+    templateVariables.put("ide", ApplicationInfo.getInstance().getFullApplicationName());
+    // Plugin version, ie. 1.1.1-10-SNAPSHOT git.c9a0e89-dirty
+    IdeaPluginDescriptor pluginDescriptor = PluginManagerCore.getPlugin(PluginId.getId("com.virtuslab.git-machete"));
+    if (pluginDescriptor != null) {
+      templateVariables.put("macheteVersion", pluginDescriptor.getVersion());
+    }
+    // Additional info about error
+    templateVariables.put("additionalInfo", additionalInfo != null ? additionalInfo : "N/A");
```

Comment:
I'd put here a sentence that would encourage the reporter to add some extra details (about the circumstances of the error, e.g. _it happened when I opened a second project in IntelliJ_)

Reply:
There was appropriate section is template but I had to delete it by accident. Good catch.

Reply:
Actually we have it in first lines of the template
```
## Bug description
Please include steps to reproduce (like `go to...`/`click on...` etc.) + expected and actual behaviour.
```
Maybe we should highlight it in some way if it's easy to miss this part?